### DISCUSSION TITLE
Feature "Keep Clean" Question

### DISCUSSION THREAD

**Discussion by dwsiddall** [YimMenu/HorseMenu#124] [Q&A] [answered]
Is "Keep Clean" meant to be for your person, or for your gun?

I got myself dirty and tried to activate this and nothing happens (I stay dirty) and I looked at my guns and their condition is not at kept getting worse the more I shot....

**Comment by Rxann** [comment, marked as answer]
> Is "Keep Clean" meant to be for your person, or for your gun?
> 
> I got myself dirty and tried to activate this and nothing happens (I stay dirty) and I looked at my guns and their condition is not at kept getting worse the more I shot....

Yeah, it is meant to be for your person but I did also notice it doesn't really work.
